bauble_macros/tests/derive.rs

```diff
@@ -55,28 +55,15 @@ fn test_flattened() {
     #[derive(FromBauble, PartialEq, Debug)]
     #[bauble(flatten)]
     enum Test {
```

Comment:
Maybe we can create a seperate test for flatten? I think it would be good to keep simple tests around too

Reply:
I also think we should have github actions to run these tests, created an issue for that #3.

Reply:
`enum`s actually don't convert currently, since `EnumVariant` isn't constructed anywhere. Syntactically, I'm not sure how we could differentiate a struct and an enum variant, so I could just remove enums from the conversion and just expect structs for the variants

Reply:
I think it's fine for now, can merge this as is then. But we should make enums without flatten match on `Value::Enum` instead